install_dependencies.R

```diff
@@ -1,4 +1,8 @@
-install.packages(c('testthat', 'vdiffr', 'spelling','covr'), repos = 'http://cran.us.r-project.org')
+install.packages(c('testthat', 'vdiffr', 'spelling','covr','remotes'), repos = 'http://cran.us.r-project.org')
 install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')
-download.file('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip?pr=false', destfile = 'ospsuite.utils.zip', mode='wb'); install.packages('ospsuite.utils.zip', repos = NULL, type = 'binary')
-download.file('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/tlf-library/artifacts/tlf.zip?pr=false', destfile = 'tlf.zip', mode='wb');  install.packages('tlf.zip', repos = NULL, type = 'binary')
+download.file('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip?pr=false', destfile = 'ospsuite.utils.zip', mode='wb')
+install.packages('ospsuite.utils.zip', repos = NULL, type = 'binary')
```

Comment:
There is currently no source package artifact without version, so I cannot use the same method as for the tlf.
Think it's ok - dependencies of ospsuite.utils would change very rarely

Reply:
Then which part of the code installs ospsuite.utils dependencies ? (purrr and R6)

https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/blob/88152af44e32e714a1d5c80237c239105ed85120/DESCRIPTION#L25-L27

Reply:
>Then which part of the code installs ospsuite.utils dependencies ? (purrr and R6)

SkyNet 😄

Reply:
![grafik](https://user-images.githubusercontent.com/25061876/231145501-20e9bf82-294c-4ead-9fa7-9a5a20567eb1.png)
which is triggered by 
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/a8d5220a1636357f962df9e6be8ae8372aeae0e6/appveyor.yml#L34

installs the dependencies of ospsuite-R
Purrr is one of the deps, R6 is probably an indirekt dependency of some packages required by the ospsuite-R

Reply:
yep, R6 is a dependency of dplyr
https://github.com/tidyverse/dplyr/blob/29307bf661a8c52b892c28afa531b38461aababb/DESCRIPTION#L32

Reply:
Ok. Even if the packages are already installed, maybe we should add their installation explicitly in this script ? 

```
# install ospsuite.utils dependencies
install.packages(c('r6', 'purrr'), repos = 'http://cran.us.r-project.org')
```

Reply:
I think we could come to the same result for TLF using `remotes::install_github("Open-Systems-Pharmacology/TLF-Library")` which installs the dependencies and also targets the latest `develop` commit.

So maybe something like this could be used for `ospsuite-rutils` ?